src/Query/TaskGroupingTree.ts

```diff
@@ -73,13 +73,13 @@ export class IntermediateTaskGroups {
         const root = new TaskGroupingTreeNode(tasks);
 
         let currentTreeLevel = [root];
-        for (const grouping of groupings) {
+        for (const grouper of groupers) {
             const nextTreeLevel = [];
             for (const currentTreeNode of currentTreeLevel) {
                 for (const task of currentTreeNode.values) {
-                    // Get properties of a task for the grouping
+                    // Get properties of a task for the grouper
                     // The returned string is rendered, so special Markdown characters will be escaped
-                    const groupNames = grouping.grouper(task);
+                    const groupNames = grouper.grouper(task);
```

Comment:
I agree to get rid of the `grouping` word, but now the `grouper.grouper` is confusing. MAybe worth to rename the `Grouper.grouper`?

Reply:
> I agree to get rid of the `grouping` word, but now the `grouper.grouper` is confusing. MAybe worth to rename the `Grouper.grouper`?

Personally, at this stage I would like to stop worrying about aspects the vocabulary of code whose classes will change a bunch more before the docs are finalised... I would instead keep going with the refactoring of the code and learn from later changes.